### PR TITLE
Add KYC matches documentSubtype

### PR DESCRIPTION
**Pull request by subenson**

### CODE REVIEW

**Review by akozhevnikov-rebilly — DISMISSED** [review]

**Comment by subenson** [comment]
@Rebilly/php please review

File: openapi/components/schemas/KycDocument/FundsMatches.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+type: object
+properties:
+  firstName:
+    description: The customer first name if it was matched, null otherwise.
+    type: string
+    example: John
+  lastName:
```
[the remark this answers is not in the document]
Comment (subenson) [reply openapi/components/schemas/KycDocument/FundsMatches.yaml:8]: Yes! Changed it for all match types

**Comment by subenson** [comment]
@Rebilly/php please review

**Review by XaosSintez — APPROVED** [review]

**Comment by subenson** [comment]
@Rebilly/php please review

**Review by VladimirBerdnik — APPROVED** [review]

**Pull request merged**